### ISSUE TITLE
Invalid LaTeX format for arcsec2

### ISSUE THREAD

**Issue by petercamps** [astropy/astropy#8699]
astropy version 3.1.2

`print (astropy.units.Unit("arcsec2").to_string("latex"))`
`$\mathrm{{}^{\prime\prime}^{2}}$`

This is invalid LaTeX because the same base string carries two exponents. The problem arises because the LaTeX string for arcsec uses an exponent to raise the primes above the baseline.

A quick fix would be to always enclose the LaTeX string for arcsec in curly braces, but the resulting layout is not so nice (the primes and the exponent are on the same level). The better fix would be to include the LaTeX string for arcsec in regular parentheses if and only if it carries an exponent.

**Comment by pllim** [comment]
> The better fix

@petercamps , thanks for reporting this. Would you be interested to open a PR with your suggested patch?

http://docs.astropy.org/en/latest/development/workflow/development_workflow.html

**Comment by mhvk** [comment]
Hmm, I think with an exponent we should indeed not use the primes at all, but rather just `$\mathrm{arcsec}^{2}$. Arguably this should happen in composite units as well (say, `pix/arcsec`). A change is not entirely trivial - the actual formatting is done in `Latex.to_string` in `units/format/latex.py`.

Note that there is already a special treatment for `arcsec` in `si.py`, in that only the un-prefixed unit is shown using primes. The same should also be done for `arcmin` and `deg`. Right now:
```
In [12]: u.mdeg.to_string('latex')
Out[12]: '$\\mathrm{m{}^{\\circ}}$'
```
(And the same for unicode output)

Possibly the simplest would be for the format dict to `def_unit` to include an extra `latex_simple` (or so) entry with the value that gets used when the unit is shown on its own.

**Comment by petercamps** [comment]
I agree with @mhvk 's suggestions. However, I can't find my way around the astropy code well enough to actually implement the change. Sorry.

**Comment by mhvk** [comment]
@petercamps - no worries, this is not a trivial part of astropy, with quite a few interlocking classes.

**Comment by pllim** [comment]
> The same should also be done for arcmin and deg

https://github.com/astropy/astropy/blob/eb20883bed9f815a0db177c4fd1474ab4077cfa1/astropy/units/si.py#L69-L75

seems to limit prefix for arcsec to "milli" and "micro". Do you mean that we should disable prefix for arcmin and deg **and** only limit them to "milli" and "micro" as well?

> include an extra latex_simple

Could you please clarify, @mhvk ? I don't see any existing implementation using "latex_simple". Will this be a completely new implementation?

**Comment by mhvk** [comment]
@pllim - I think for `arcmin` we might want to eliminate all prefixes, but for degrees we probably want all of them. Separately, we need to do something to ensure that those angular units (as well as hourangle) get presented right - the `latex_simple` would indeed be a new implementation - but there are obviously other options. E.g., one could allow the latex entry be a tuple, where the first is only used when the unit is on its own. 

As yet another alternative, we could just follow standards and not use the degree symbol, etc., by default at all, but only use it for special cases such as in `Angle`.
